packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -143,8 +148,8 @@ export const makeZoeSeatAdminFactory = baggage => {
           // doExit(), which ensures that finalPayouts() has set state.payouts.
           return E.when(
             state.subscriber.subscribeAfter(),
-            () => state.payouts[keyword],
-            () => state.payouts[keyword],
+            () => getPayoutOrThrow(state, keyword),
```

Comment:
Please use NonNullish() instead of a new function

Reply:
When switching to `NonNullish`, please remember to use the second optional `optDetails` argument so the error message remains informative.